### PR TITLE
Polymer 3 Conversion

### PR DESCRIPTION
**Pull request by dbatiste**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BrightspaceUI/typography.git"
+  },
+  "name": "d2l-typography",
   "scripts": {
     "build:sass": "node-sass --output-style expanded ./test/acceptance/typography.css.scss ./test/acceptance/typography.css",
-    "postinstall": "polymer install --variants",
-    "lint": "polymer lint -i *.html",
+    "lint": "npm run lint:wc && npm run lint:js",
```
Comment (awikkerink) [inline package.json]: `lint:wc` doesn't exist

Comment (dbatiste) [reply package.json]: ya.  do we want to clean this up (make consistent) as we go?  alternative looks like the script would need to be a little more flexible.

Comment (svanherk) [reply package.json]: My original thinking was that we'd take this opportunity to make things consistent (make the `package.json`s all use the same script conventions before running the script)

Comment (dbatiste) [reply package.json]: Sounds good / agree.

**Pull request closed without merging**